webpack.config.js

```diff
@@ -90,7 +90,8 @@ module.exports = function makeConfig(grunt) {
                 {
                     test: /\.jsx?$/,
                     loader: 'eslint-loader',
-                    exclude: shouldExclude
+                    // superdesk apps handle their own linter
+                    exclude: (p) => shouldExclude(p) || sdConfig.apps.some(app => p.indexOf(app) > -1),
```

Comment:
I find my solution more maintainable as it reuses an exclusion function we already defined and it doesn't use an hard coded value to search for.
And sorry but yes, I am looking for a working solution, at least until you suggest a working one. Yours doesn't do what I'm expecting for (probably because of symbolic links created by `npm link`).